stdlib/3/os/__init__.pyi

```diff
@@ -17,75 +18,72 @@ _T = TypeVar('_T')
 
 # ----- os variables -----
 
-if sys.version_info >= (3, 2):
```

Comment:
I would actually like to keep these in order to make it possible to merge stdlib/2/ and stdlib/3/ in the future.

Reply:
Only in os or in other modules as well?

Reply:
Only os.